test/performance/odp_sched_latency.c

```diff
@@ -764,24 +896,27 @@ int main(int argc, char *argv[])
 	(void)odp_cpumask_to_str(&cpumask, cpumaskstr, sizeof(cpumaskstr));
 
 	printf("Test options:\n");
-	printf("  Worker threads: %i\n", num_workers);
-	printf("  First CPU:      %i\n", odp_cpumask_first(&cpumask));
-	printf("  CPU mask:       %s\n", cpumaskstr);
-	printf("  Test rounds:    %iM\n", args.test_rounds);
-	printf("  Warm-up rounds: %i\n", args.warm_up_rounds);
+	printf("  Worker threads:   %i\n", num_workers);
+	printf("  First CPU:        %i\n", odp_cpumask_first(&cpumask));
+	printf("  CPU mask:         %s\n", cpumaskstr);
+	printf("  Test rounds:      %iM\n", args.test_rounds);
+	printf("  Warm-up rounds:   %i\n", args.warm_up_rounds);
+	printf("  Isolated groups:  %i\n", args.isolate);
+	printf("  Number of groups: %i\n", num_group);
+	printf("  Created groups:   %i\n", tot_group);
 	printf("\n");
 
 	thread_tbl = calloc(sizeof(odph_thread_t), num_workers);
 	if (!thread_tbl) {
 		ODPH_ERR("no memory for thread_tbl\n");
-		return -1;
+		exit(EXIT_FAILURE);
 	}
 
-	shm = odp_shm_reserve("test_globals",
-			      sizeof(test_globals_t), ODP_CACHE_LINE_SIZE, 0);
+	shm = odp_shm_reserve("test_globals", sizeof(test_globals_t), ODP_CACHE_LINE_SIZE, 0);
 	if (shm == ODP_SHM_INVALID) {
 		ODPH_ERR("Shared memory reserve failed.\n");
-		return -1;
+		err = -1;
```

Comment:
`thread_tbl` is not freed in case of failures. Probably no need to allocate it from heap in the first place.

Reply:
Fixed in v2. Change to allocate thread_tbl from stack.